### PR TITLE
added more components

### PR DESCRIPTION
**Pull request by Rami-ismail**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // import { TornadoDragon } from "./mockData";
+import CardsDetail from "./components/CardDetails/CardsDetail";
 import CardsList from "./components/CardList/CardsList";
+import { TornadoDragon } from "./mockData";
```
Comment (Rami-ismail) [inline src/App.tsx:4]: good job

Comment (Rami-ismail) [reply src/App.tsx:4]: we can refactor this further

**Review by ahmadhadwan — COMMENTED** [review]
10/10

**Review by ahmadhadwan — APPROVED** [review]
approved

**Pull request merged**